packages/cli/openapi-parser/src/v3/converters/example/ExampleEndpointFactory.ts

```diff
@@ -31,6 +31,10 @@ export class ExampleEndpointFactory {
             responseSchemaIdResponse != null
                 ? this.exampleTypeFactory.buildExample(responseSchemaIdResponse.schemaInstanceId)
                 : undefined;
+        if (requestExample === undefined && responseExample === undefined) {
+            return undefined;
+        }
```

Comment:
@dsinghvi This simplifies some of the error handling in the OpenAPI IR -> Fern IR translation. Otherwise we'd need to verify that the example produced should actually be rendered. The Fern IR converter should always assume any example its given is valid / should be written.

Reply:
thanks for adding, makes sense